conf/test.config

```diff
@@ -25,5 +25,5 @@ params {
     // Input data
     // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
     // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = params.pipelines_testdata_base_path + 'cellpainting/minimal_dataset/cpg0016-jump/source_4/workspace/load_data_csv/2021_04_26_Batch1/BR00117035/load_data.csv'
+    input = "${projectDir}/assets/samplesheet.csv"
```

Comment:
Since the test data is in a public s3 bucket, we might want to consider adding this to the test profile configuration so that it works for people with aws creds setup locally out of the box. Otherwise there is a chance people testing locally might run into TOKEN issues with their aws setup. This should ensure that the s3 request is always anonymous and always works...

```
aws {
    client {
        anonymous = true
    }
}

```

Reply:
Actually, we shouldn't do this because it would break access to user specific resources: https://github.com/nf-core/configs/pull/549. Leaving this comment here though in case someone tries to run the test profile locally and runs into S3 errors due to token or sign requests!